packages/botonic-nlu/src/prediction.js

```diff
@@ -0,0 +1,29 @@
+import * as tf from '@tensorflow/tfjs'
+import { Tokenizer, padSequences } from './preprocessing'
+import { getEntities } from './ner'
+
+export function getPrediction(input, model, nluData) {
+  let tokenizer = new Tokenizer(nluData.vocabulary)
+  let sequence = tokenizer.samplesToSequences(input)[0]
```

Comment:
why only [0]?

Reply:
when preprocessing the entire set of example utterances in the training phase, this method accepts a list of samples and returns a list as well, so then, when we use it for predict over only one "input"(userInput), we only need one sequence